packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -263,15 +264,17 @@ export const executeTxn = async (
   senderAddress: string,
   privateKey: string | Uint8Array,
   txnInvocation: Call | Call[],
-  abis?: Abi[],
+  transactionVersion: TransactionVersion = TRANSACTION_VERSION,
```

Comment:
```suggestion
```

it is setting the default version everywhere, try to make it single source of truth